mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafText.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  LeafText.swift
+//  
+//
+//  Created by Cris Messias on 06/10/23.
+//
+
+import SwiftUI
+
+//MARK: - TitleModifier
+
+public struct TitleModifier: ViewModifier {
+    @Environment(\.leafTheme) private var theme
+    public func body(content: Content) -> some View {
+        content
+            .font(theme.font.titleLarge)
+            .bold()
+            .lineSpacing(0.2)
```

Comment:
This value 0.2 could also be part of our theme. What do you think?

Reply:
done and dusted!